apps/examples/wc-vanilla/main.js

```diff
@@ -5,6 +5,7 @@ document.querySelector('#app').innerHTML = `
     <div>
         <h1>Hello World!</h1>
         <pie-button id="counter" type="button"></pie-button>
+        <pie-icon-button size='medium' type='button' variant='primary'></pie-icon-button>
```

Comment:
Nit: I think we should use double quotes inside HTML?
```suggestion
        <pie-icon-button size="medium" type="button" variant="primary"></pie-icon-button>
```

Reply:
Fixed